### PR TITLE
Api mappings

### PR DESCRIPTION
**Pull request by didaRatsimba**
wanted to work on #49 but tests failed so I corrected test first

### CODE REVIEW

File: src/main/java/com/projecty/projectyweb/task/TaskController.java [hunk 1]
```diff
@@ -83,10 +83,10 @@ public void addTaskPost(
         return map;
     }
 
-    @PostMapping("deleteTask")
+    @PostMapping({"deleteTask/project/{projectId}/task/{taskId}", "deleteTask/task/{taskId}", "deleteTask/project/task/{taskId}"})
```
Comment (fokion) [inline src/main/java/com/projecty/projectyweb/task/TaskController.java]: This can even be a DELETE action with a `@DeleteMapping`

Comment (didaRatsimba) [reply src/main/java/com/projecty/projectyweb/task/TaskController.java]: I changed @PostMapping to @DeleteMapping for this method and replaced left @RequestParam with @PathVariable

**Comment by marcinadd** [comment]
Oh thanks for your effort but I have probably finished this already. I used Gson for testing. @fokion Can you take a look? What do you think about mappings? Is better by approach or @didaRatsimba to URLs?

**Comment by fokion** [comment]
@marcinadd  what do you want me to have a look?  do you have a branch that you can share ?

**Comment by marcinadd** [comment]
@fokion  I didn't know that somebody had started working on mappings so I started work myself.
Now there are a lot of merge conflicts. I want you to look at my and @didaRatsimba api-mappings branches and tell which looks better for you.

**Comment by marcinadd** [comment]
Thanks for your contribution. I was hard to decide but listening to some opinions I've decided to use my simpler mappings.

**Comment by didaRatsimba** [comment]
Okay, I will ask before taking a task then.

**Pull request closed without merging**